components/builder-web/app/actions/projects.ts

```diff
@@ -120,45 +120,43 @@ export function fetchProjects(token: string) {
   };
 }
 
-export function deleteProject(id: string, token: string, origin: string) {
+export function deleteProject(id: string, token: string) {
   return dispatch => {
     new BuilderApiClient(token).deleteProject(id).then(response => {
-      dispatch(resetProjectHint());
-      dispatch(requestRoute(["/origins", origin]));
       dispatch(addNotification({
-        title: "Project deleted",
+        title: "Plan deleted",
```

Comment:
I'm not sure whether this nomenclature might be confusing to the user - maybe we should say "Plan link deleted", since we're not doing anything to the actual plan itself.  Not a blocker for merge tho.

Reply:
Ah, that's a good point. Easy enough to clarify that, and much less potentially terrifying. :smile: